cpp/foxglove/include/foxglove/mcap.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <memory>
+#include <string>
+
+struct foxglove_mcap_writer;
+
+namespace foxglove {
+
+struct McapWriterOptions {
+  std::string_view path;
+  std::string_view profile;
+  std::string_view library;
+  uint64_t chunkSize;
+  bool useChunks;
```

Comment:
I would recommend adding inline defaults for all the primitive types, iirc this helps avoid situations where we might accidentally read from uninitialized memory (plus it helps users to set sane default values if they don't care to change them)
```suggestion
  uint64_t chunkSize = 1024*768;
  bool useChunks = true;
```